code/modules/mob/living/carbon/human/life.dm

```diff
@@ -296,18 +295,76 @@
 			return TRUE
 	return ..()
 
+/// Handles heart attacks, even if we don't have a heart
 /mob/living/carbon/human/proc/handle_heart(seconds_per_tick, times_fired)
-	var/we_breath = !HAS_TRAIT_FROM(src, TRAIT_NOBREATH, SPECIES_TRAIT)
```

Comment:
I think it's worth noting that all nobreath sources will now prevent falling unconscious from a heart attack now rather than just from your species (for the changelog?)

Reply:
Which I'll be real, it's weird that they didn't. I'll add that to the changelog.